### PR TITLE
Fix useFetchMember hooks

### PR DESCRIPTION
**Pull request by masskoder**
### Scope of changes

With this fixes, user should be able to fetch member detail endpoints

### Type of change

- [x] bug fix
- [ ] new feature
- [ ] documentation
- [ ] other (describe)

### Author checklist

- [ ] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ]  I have updated the dependencies list
- [ ]  I have recompiled and included new protocol buffers to reflect changes I made
- [ ]  I have added new test fixtures as needed to support added tests
- [x]   Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [x]  I have moved the associated Shortcut story to "Ready for Review"

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.

### CODE REVIEW

File: web/gds-user-ui/src/modules/dashboard/member/components/MemberModal/MemberModal.tsx [hunk 1]
```diff
@@ -19,28 +19,36 @@ import { useFetchMember } from '../../hooks/useFetchMember';
...
   const network = useSelector(memberSelector).members.network;
-  const { member, isFetchingMember } = useFetchMember({ vaspId: memberId, network });
+  const { member, isFetchingMember, error, wasMemberFetched } = useFetchMember({
+    vaspId: memberId,
+    network
+  });
+
+  if (error) {
+    console.log('[] error', error?.response?.data);
+    // close the modal for now , later we need to show a toast message
```
Comment (daniellemaxwell) [inline web/gds-user-ui/src/modules/dashboard/member/components/MemberModal/MemberModal.tsx:37]: Would it be possible to return a toast error with the release?

File: web/gds-user-ui/src/modules/dashboard/member/hooks/useExportMembers.tsx [hunk 2]
```diff
@@ -17,8 +22,8 @@ const useExportMembers = () => {
         downloadMembers2CVS(members?.vasps);
         setIsLoading(false);
       }, LOADING_TIMEOUT);
-    } catch (error) {
-      console.log('[useExportMembers] error: ', error);
+    } catch (er: any) {
```
Comment (daniellemaxwell) [inline web/gds-user-ui/src/modules/dashboard/member/hooks/useExportMembers.tsx:25]: Should `er` be changed to `err`?

File: web/gds-user-ui/src/modules/dashboard/member/hooks/useExportMembers.tsx [hunk 3]
```diff
@@ -6,9 +6,14 @@ import { memberSelector } from '../member.slice';
...
 const useExportMembers = () => {
   const network = useSelector(memberSelector).members.network;
-  const { members, getMembers } = useFetchMembers(network);
+  const { members, getMembers, error } = useFetchMembers(network);
   const [isLoading, setIsLoading] = useState<boolean>(false);
   const LOADING_TIMEOUT = 500;
+
+  const isUnverified = error && error?.response?.status === 451;
+  const isMemberEmpty = !members?.vasps?.length;
+  const hasError = !isUnverified && error; // if error is not 451, then it's a real error
```
Comment (daniellemaxwell) [inline web/gds-user-ui/src/modules/dashboard/member/hooks/useExportMembers.tsx:15]: Thanks for adding this!

**Review by daniellemaxwell — APPROVED** [review]
Adding the ability to display toast errors and making the `Export` button disabled will be be really helpful!

File: web/gds-user-ui/src/modules/dashboard/member/components/MemberModal/MemberModal.tsx (hunk 1 above)
Comment (masskoder) [reply web/gds-user-ui/src/modules/dashboard/member/components/MemberModal/MemberModal.tsx:37]: A story have been already created to Add it in a different PR.

**Pull request merged**